packages/@react-stately/collections/docs/collections.mdx

```diff
@@ -0,0 +1,387 @@
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-stately/collections';
+import dataDocs from 'docs:@react-stately/data';
+import {HeaderInfo, PropTable, TypeLink} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+---
+category: Collections
+---
+
+# Collection Components
+
+Many components display a collection of items, and provide functionality such as keyboard 
+navigation, selection, and more. React Spectrum has a unified API to define the items
+displayed in these components.
+
+## Introduction
+
+There are many components that display a collection of items of some kind. For example, lists, menus, selects,
+tables, trees, and grids. These collections can usually be navigated with the keyboard using arrow keys, and
+many have some form of selection. Many support loading data asynchronously, updating that data over time,
+virtualized scrolling for performance with large collections, and more.
+
+There are many ways one could design an API for components like this: JSX children, a prop that accepts a
+list of option objects, as a datasource object with getter functions for each item, etc.. Selection, and 
+other states like disabled, could be passed as options to each item, or as a separate top-level prop.
+Each of these has various tradeoffs. This page describes how we do it in React Spectrum, and covers some
+of these tradeoffs in detail.
+
+## Design goals
+
+Our main design goal was to provide a consistent API across many types of collection components that is
+easy to learn, performant on large collections, and extensible for advanced features. Some of the usecases
+we considered were:
+
+- Static data
+- Dynamic data
+- Async loading
+- Sections/groups of data from a single source
+- Sections from different sources
+- Single and multiple selection (controlled and uncontrolled)
+- Controlled and uncontrolled expanded states for components like trees and accordions
+- Marking items as selected, expanded, disabled, etc. prior to loading them from a server
+- Drag and drop of items
+- Virtualization for large collections
+- Infinite scrolling to load more items on demand
+
+## Static collections
+
+React Spectrum implements a JSX-based API for defining collections. This allows an intuitive way to provide
+items, which can contain rich rendering of item contents, and various options as props. Building heirarchical
```

Comment:
*which can contain rich content rendering and various ...
having "item" twice is confusing.